src/elements/Links.js

```diff
@@ -0,0 +1,134 @@
+import { Grid, Button, makeStyles } from "@material-ui/core";
+import React from "react";
+import { useKeyPressAction } from "../hooks/useKeyPressAction";
+
+const useStyles = makeStyles((theme) => ({
+  grid: {
+    textAlign: "center",
+  },
+  shortcut: {
+    position: "absolute",
+    right: 5,
+    top: 1,
+    fontSize: "0.6rem",
+  },
+}));
+
+const directionals = ["north", "south", "east", "west"];
+const splitLinks = (links = []) => {
+  const isDirectional = (link) =>
+    Boolean(
+      directionals.find((d) => d.toLowerCase() === link.name.toLowerCase())
+    );
+  const directionalLinks =
+    links
+      .filter((link) => isDirectional(link))
+      .reduce((dir, link) => {
+        dir[link.name.toLowerCase()] = link;
+        return dir;
+      }, {}) || {};
+  const otherLinks = links.filter((link) => !isDirectional(link)) || [];
+
+  return { directionalLinks, otherLinks };
+};
+
+const outputLink = (link, onClick, altText = "", customOpts = {}) => {
```

Comment:
pull `altText` (and `onClick`? and `link`?) into `customOpts` to avoid signature of doom.